packages/pigeon/pigeons/core_tests.dart

```diff
@@ -12,22 +12,45 @@ enum AnEnum {
 
 // A class containing all supported types.
 class AllTypes {
-  bool? aBool;
-  int? anInt;
-  double? aDouble;
-  String? aString;
-  Uint8List? aByteArray;
-  Int32List? a4ByteArray;
-  Int64List? a8ByteArray;
-  Float64List? aFloatArray;
+  AllTypes(
+    this.aBool,
+    this.anInt,
+    this.aDouble,
+    this.aString,
+    // TODO(stuartmorgan): Enable these once they work for all generators;
+    // currently at least Swift is broken.
+    // See https://github.com/flutter/flutter/issues/115906
+    // this.aByteArray,
+    // this.a4ByteArray,
+    // this.a8ByteArray,
+    // this.aFloatArray,
+    this.aList,
+    this.aMap,
+    this.nestedList,
+    this.mapWithAnnotations,
+    this.mapWithObject,
+    this.anEnum,
```

Comment:
You can give these all default values here to make the Dart instantiation less painful.